forward_engineering/api.js

```diff
@@ -90,15 +99,11 @@ module.exports = {
 		}
 	},
 
-	async applyToInstance(data, logger, callback, app) {
-		const NOT_FOUND_RESPONSE_CODE = 'NotFoundException';
-		const BAD_REQUEST_RESPONSE_CODE = 'BadRequestException';
-		const NOTHING_TO_UPDATE_RESPONSE_MESSAGE = 'Invalid request. Please provide at least one field to update.';
+	async applyToInstance(data, logger, callback) {
 		if (!data.script) {
 			return callback({ message: 'Empty script' });
 		}
 
-		logger.clear();
```

Comment:
We clear the log file before processing the "Apply to instance" in all plugins. Not sure that we should skip this step here now. At least we will have different behavior between plugins unless clearing the log here doesn't make sense and it done before in the logic